Repo-Integration/Self-Managed/README.md

```diff
@@ -67,24 +67,66 @@ Execution instructions:
+
+```shell
+export ws_key='your activation key between single quotes'
+export github_com_token='replace-with-your-github-token-inside-single-quotes'
+export graylog_root_password='the password you would like to use to login to graylog'
+```
 
-```export ws_key='your activation key between single quotes'```
-
-- Add your Github.com access token. This is important for Remediate and Renovate to prevent Gitbub rate-limiting imposed on non-authenticated requests.
-
-`export GITHUB_COM_TOKEN='replace-with-your-github-token-inside-single-quotes'`
-  
 - Run the setup.sh script for your appropriate source control management system as shown in options above
 
-- Run docker compose depending on how it was installed. Options defined -
-  - SCA only  - docker-compose.yaml
-  - SCA and SAST - docker-compose-sast.yaml
-    - **Note: this is currently only supported for GHE** [(a dedicated SAST scanner container)](https://docs.mend.io/bundle/integrations/page/deploy_with_docker.html#Target-Machine:-Run-the-Containers).
+- Run the following commands to increase your memory map count for graylog's elasticsearch.
 
-```docker compose -f <compose file> up```
+```shell
+sudo sh -c 'echo "vm.max_map_count=262144" >> /etc/sysctl.conf'
+sudo sysctl -p
+```
 
-- Run docker compose in detached mode depending on how it was installed.
+**NOTE: Please remove the following lines from the docker-compose file if you are running on Windows with Docker Desktop**
+```
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+```
+
+- Run docker compose in detached mode for your desired setup. Options defined -
+  - SCA only ```docker compose up -d```
+  - SCA and SAST ```docker compose -f docker-compose-sast.yaml up -d```
+    - **Note: this is currently only supported for GHE** [(a dedicated SAST scanner container)](https://docs.mend.io/bundle/integrations/page/deploy_with_docker.html#Target-Machine:-Run-the-Containers).
 
-```docker compose -f <compose file> up -d```
+- After running this, wait until all containers are created.  Do not be concerned if the self-managed-graylog container has errored out as unhealthy.  This will occur until the manual setup below been performed.
+  - Run `docker compose logs --follow` in a terminal to get the username and password for first time login
+  - Navigate to http://your-host-ip-address:9000 and log in with username: `admin` and password: `the password shown in the graylog logs`
+  - Follow the setup steps and keep all of the defaults.
+  - After clicking resume setup all containers should be created and healthy and Graylog will automatically install the Mend Content Pack and start accepting input from the integrations which will also start. 
+  - Log into the platform with with the username: admin and the password you set in `$graylog_root_password`
+
+**TODO** Add Steps to view Dashboard to ensure integration is running smoothly
```

Comment:
Doesn't need to be a dashboard, just add steps for a user to search in Graylog to verify that all is running well.  Mostly the table is what is important.

Reply:
Done.